tests/fixtures/points.py

```diff
@@ -37,10 +39,17 @@ def generate_records(
         if random_ids:
             idx = str(uuid.uuid4())
 
+        vectors = random_vectors(vector_sizes)
+
+        if skip_vectors:
+            if random.random() > 0.8:
+                vector_to_skip = random.choice(list(vectors.keys()))
```

Comment:
do we want to fail with exception if someone set `vector_sizes = int` and `skip_vectors = True` ?